lib/analyzer.js

```diff
@@ -138,7 +138,7 @@ function analyzer(fileLocation, rules) {
       let ext = extractExtension(fileLocation);
 
       let templateFindings = [];
-      if (fileLocation.match('\.template\.js$')) { // Dispatch to template searcher
+      if (fileLocation.match('.template.js')) { // Dispatch to template searcher
```

Comment:
missed `endsWith` here

Reply:
That's a careless mistake. Good catch! 
Fixed in next [commit](https://github.com/mapbox/secret-shield/pull/30/commits/e17723a4cf121cbdf9245fcc2774f9daf935cf07)